app/models/concerns/slugifiable.rb

```diff
@@ -0,0 +1,14 @@
+module Sluggable
+  module InstanceMethods
+    def slug
+   
+      username.downcase.gsub(" ", "-")
```

Comment:
This is interesting...wouldn't it be possible to have multiple users with the same username and/or slug? For example '@john smith' and '@john-smith'.